app/components/booking_bug_api.rb

```diff
@@ -0,0 +1,12 @@
+class BookingBugAPI
```

Comment:
Your right, this object is potentially not a component, but with only one of them I am not sure what it is. or if it is even needed, similarly with the BookingBugConnection it still needs a proper home. 

I've moved them both here as think it is the best location for now, with the understanding that this should change as more are built in a similar way to how importers and components have been extracted.

Happy for you to move these/suggest a location if you have a clear visible at this stage.